### DISCUSSION TITLE
Hacktoberfest Participation and API Endpoints

### DISCUSSION THREAD

**Discussion by mdleiton** [fief-dev/fief#449] [General]
Hi,
Will this project be participating in Hacktoberfest? I’d love to contribute!
I’m also interested in adding new API endpoints, such as retrieving users by role. Let me know if that would be helpful.

**Comment by fief-bailiff** [comment]
Hail, @mdleiton 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!
